plugins/woocommerce/tests/e2e-pw/tests/merchant/create-simple-product.spec.js

```diff
@@ -4,9 +4,9 @@ const wcApi = require( '@woocommerce/woocommerce-rest-api' ).default;
 const virtualProductName = 'Virtual Product Name';
 const nonVirtualProductName = 'Non Virtual Product Name';
 const productPrice = '9.99';
-let shippingZoneId, virtualProductPermalink, nonVirtualProductPermalink;
+let shippingZoneId, virtualProductId, nonVirtualProductId;
 
-test.describe( 'Add New Simple Product Page', () => {
+test.describe.serial( 'Add New Simple Product Page', () => {
```

Comment:
I had to use `serial` here because these two tests depend on the test immediately before them:
- `can have a shopper add the simple virtual product to the cart`
- `can have a shopper add the simple non-virtual product to the cart`

Without `serial`, the tests they depend on will not be re-executed during the retry, thus the retry will certainly fail. Using `serial` will make sure the tests they depend on will be re-executed during the retry even though they have already passed in the initial run.